src/cooler/util.py

```diff
@@ -204,56 +205,82 @@ def argnatsort(array: Iterable[str]) -> np.ndarray:
     cols = tuple(zip(*(natsort_key(x) for x in array)))
     return np.lexsort(cols[::-1])
 
-
 def read_chromsizes(
-    filepath_or: str | IO[str],
+    filepath_or: str | io.StringIO,
     name_patterns: tuple[str, ...] = (r"^chr[0-9]+$", r"^chr[XY]$", r"^chrM$"),
     all_names: bool = False,
+    verbose: bool = False,  # Optional parameter to enable verbose output
```

Comment:
There is little use for a "verbose" mode if a function isn't doing much and isn't running for long periods of time.